### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by aryehlev**
its still not 100% as after i choose products and then a city the button does not enable till a new action was taken

### CODE REVIEW

**Comment by aryehlev** [comment]
i want to link this PR to an issue, does anyone know how i can do this?

**Comment by simzacks** [comment]
Linking a PR to an issue:
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

File: superx/routes/home.py [hunk 1]
```diff
@@ -115,7 +115,12 @@ def add_item():
     cart_list.append(item)
     session['cart'] = cart_list
 
-    return ''
+    was_city_chosen = False
+    if 'city' in session:
+        was_city_chosen = True
```
Comment (simzacks) [inline superx/routes/home.py]: Is it possible that session["city"] == None or ""?
If so, you might want to use:
```
if session.get("city") 
# the default of get is None, if it doesn't exist.
```

You can also cut these 3 lines to:
`was_city_chosen = True if 'city' in session else False`

File: superx/routes/home.py [hunk 2]
```diff
@@ -115,7 +115,10 @@ def add_item():
     cart_list.append(item)
     session['cart'] = cart_list
 
-    return ''
+    was_city_chosen = bool('city' in session)
```
Comment (simzacks) [inline superx/routes/home.py:118]: FYI - no need for the bool function.
this is the same as 
was_city_chosen = 'city' in session

**Review by simzacks — APPROVED** [review]

**Pull request merged**